page/src/components/EventCount/EventCount.jsx

```diff
@@ -0,0 +1,12 @@
+const getCountText = (count) => {
+    console.log(`count: ${count}`);
```

Comment:
console.log are only used in dev/debug phase, isn't it?
Can you remove them please? :)